docfx.json

```diff
@@ -907,10 +907,21 @@
             },
             "ms.update-cycle": {
                 "docs/ai/**/**.{md,yml}": "180-days",
-                "docs/csharp/whats-new/**/**.{md,yml}": "1825-days",
-                "docs/framework/**/**.{md,yml}": "3650-days",
+                "docs/csharp/whats-new/**/**.{md,yml}": "1095-days",
+                "docs/csharp/advanced-topics/interop/**/**.{md,yml}": "1825-days",
+                "docs/csharp/advanced-topics/reflection-and-attributes/**/**.{md,yml}": "1825-days",
+                "docs/csharp/fundamentals/exceptions/**/**.{md,yml}": "1825-days",
+                "docs/csharp/language-reference/**/**.{md,yml}": "1095-days",
+                "docs/csharp/language-reference/compiler-messages/**/**.{md,yml}": "3650-days",
+                "docs/csharp/language-reference/xmldoc/**/**.{md,yml}": "3650-days",
+                "docs/csharp/linq/**/**.{md,yml}": "1825-days",
+                "docs/csharp/misc/**/**.{md,yml}": "3650-days",
+                "docs/csharp/roslyn-sdk/**/**.{md,yml}": "1825-days",
+                "docs/csharp/specification/**/**.{md,yml}": "1825-days",
+                "docs/framework/**/*.md": "1825-days",
                 "docs/iot/**/**.{md,yml}": "1825-days",
-                "docs/visual-basic/**/**.{md,yml}": "3650-days"
+                "docs/framework/additional-apis/pos-for-net/**/**.md": "1825-days",
```

Comment:
Delete this line since it's the same tier as its parent /framework/ folder and therefore doesn't need a special callout.